velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -583,19 +583,21 @@ ExpressionFuzzer::ExpressionFuzzer(
   LOG(INFO) << fmt::format(
       "Functions with at least one supported signature: {} ({:.2f}%)",
       supportedFunctions_.size(),
-      (double)supportedFunctions_.size() / totalFunctions * 100);
+      static_cast<double>(supportedFunctions_.size() / totalFunctions * 100));
```

Comment:
This should be `static_cast<double>(supportedFunctions_.size()) / totalFunctions * 100`.
Same below.

Reply:
Nice catch. Thanks !